src/controller/DevicePairingDelegate.h

```diff
@@ -130,13 +130,24 @@ class DLL_EXPORT DevicePairingDelegate
     virtual void OnICDRegistrationInfoRequired() {}
 
     /**
-     * @bried
+     * @brief
      *   Called when the registration flow for the ICD completes.
      *
      * @param[in] icdNodeId    The node id of the ICD.
      * @param[in] icdCounter   The ICD Counter received from the device.
      */
     virtual void OnICDRegistrationComplete(NodeId icdNodeId, uint32_t icdCounter) {}
+
+    /**
+     * @brief
+     *   Called when ICDStayActiveDuration is set and the stay active response is received in the end of the LIT ICD commissioning
```

Comment:
"of the ... flow completes" needs to be fixed grammar-wise.

Reply:
updated, thanks